frontend/pages/model/[uuid]/new-version.tsx

```diff
@@ -183,7 +184,12 @@ function Upload() {
 
   return (
     <Paper variant='outlined' sx={{ my: { xs: 3, md: 6 }, p: { xs: 2, md: 3 } }}>
-      <SubmissionError error={error} />
+      <MessageAlert
+        message={error}
+        severity='error'
+        linkText='More info'
+        href='/docs/users/upload-a-model/updating-a-model'
```

Comment:
The URL should not be hardcoded, as there may be different errors.  It should instead parse the URL from the error repsonse.